packages/data-stores/src/plans/resolvers.ts

```diff
@@ -110,7 +110,7 @@ function normalizePlanProducts(
 			periodAgnosticSlug: periodAgnosticPlan.periodAgnosticSlug,
 			storeSlug: planProduct.product_slug,
 			rawPrice: planProduct.raw_price,
-			pathSlug: planProduct.path_slug,
+			pathSlug: planProduct.product_slug === 'free_plan' ? 'free' : planProduct.path_slug,
```

Comment:
Ok to use the `'free'` string directly here since the response is typed: https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/plans/types.ts#L34